cinema/models.py

```diff
@@ -1,6 +1,12 @@
+import os
+import pathlib
+import uuid
+
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.conf import settings
+from django.utils.text import slugify
+from pip._vendor.msgpack import ext
```

Comment:
The import from `pip._vendor.msgpack` seems unnecessary for this file. Consider removing it if it's not being used.